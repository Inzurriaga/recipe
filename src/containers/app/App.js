import React from 'react';
import Header from "../header/Header"
import RecipeContainer from "../recipeContainer/RecipeContainer"

function App() {
  return (
    <div className="App">
      <Header></Header>
      <RecipeContainer></RecipeContainer>
    </div>
  );
}

export default App;
