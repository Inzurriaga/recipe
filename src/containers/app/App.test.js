import React from 'react';
import { App } from './App';
import { shallow } from "enzyme";

const mockSetRecipes = jest.fn();

describe("App", () => {

  describe("App", () => {
    let wrapper;

    beforeEach(() => {
      wrapper = shallow(
        <App setRecipes={mockSetRecipes}/>
      );
    })

    it("should match snap shot", () => {
      expect(wrapper).toMatchSnapshot();
    })
  })

  describe("mapDispatchToProps", () => {

  })
})
