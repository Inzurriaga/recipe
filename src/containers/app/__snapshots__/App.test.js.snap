// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App App should match snap shot 1`] = `ShallowWrapper {}`;
